README.md

```diff
@@ -1,3 +1,149 @@
+
+## Configuring Code Climate
+
+1. After adding the project to CC, go to `Repo Settings`
+2. On the `Test Coverage` tab, copy the `Test Reporter ID`
+3. Set the copied value as environment variable `CC_TEST_REPORTER_ID` (and repo Secrets)
+
+## Sonarqube Integration
+
+1. Log into Sonarqube console (`SONAR_URL`) and create new Project key(`SONAR_PROJECT`)
+2. Generate a token for the new project and copy
+3. Set the token value as environment variable `SONAR_TOKEN`
+
+### Usage
+
+```
+sonar-scanner \
+  -Dsonar.qualitygate.wait=true \
+  -Dsonar.host.url=$SONAR_URL \
+  -Dsonar.login=$SONAR_TOKEN \
+  -Dsonar.projectKey=$SONAR_PROJECT \
+  -Dsonar.scm.provider=git \
+  -Dsonar.java.binaries=/tmp \
+  -Dsonar.nodejs.executable=$(which node) \
+  -Dsonar.projectVersion=$(echo $GITHUB_SHA | cut -c1-8) \
+  -Dsonar.sources=. \
+  -Dsonar.projectBaseDir=. \
+  -Dsonar.javascript.lcov.reportPaths=coverage/lcov.info
+```
+
+## Build Android Release
+
+### Configuration
+
+🚧 UNDER CONSTRUCTION 🚧
+
+### Create release
+
+🚧 UNDER CONSTRUCTION 🚧
+
+## Build iOS Release
+
+1. Make sure that the version was already bumped if it applies.
+2. Select on Xcode the scheme of the env you want to create the release for.
+3. For the device select **generic iOS device**.
+4. Then go to **Product** -> **Archive**.
+5. After it is done processing and the archive succeeds the **organizer** will open. Here is where you can see all the previously generated archives.
+
+## Managing multiple environments
+
+The base is already equipped with three main environments: `dev`, `staging`, `production`. All the env files you need to provide are as follows: `.env`, `.env.staging`, and `.env.prod`.
+
+If you want to add a new env here are the steps to follow:
+
+1. Create a new env file with the format `.env.{name}`.
+
+### Android
+
+2. go to `android/app/build.gradle` and add the env file association to `project.ext.envConfigFiles` following the existing ones as example.
+3. Scroll down to the `flavors` section and add a new flavor with the following format:
+
+```
+{name} {
+  applicationIdSuffix '.{name}'
+  minSdkVersion rootProject.ext.minSdkVersion
+  targetSdkVersion rootProject.ext.targetSdkVersion
+}
+```
+
+4. Inside `android/app/src` copy one of the existing env folders like `staging` and rename it with the name of your new flavor.
+5. In that new folder you will see a folder called `values` and inside a file called `strings.xml` there you can set the app name that is going to appear for this flavor. You can also set special app icons for each flavor inside `res` folder.
+6. (optional) go ahead and add new scripts in the `package.json` file for this new env. As you can see, the other envs already have scripts to run, build and build release, this will make your development workflow a lot easier.
```

Comment:
If we have the bump script maybe we could also add an optional step of adding the new environment there, so it can have the version bump easier.